bin/liquidator/src/rebalance.rs

```diff
@@ -266,6 +290,45 @@ impl Rebalancer {
         Ok((in_token.mint, in_amount))
     }
 
+    fn can_use_sanctum_for_token(&self, mint: Pubkey) -> anyhow::Result<bool> {
+        if !self.config.use_sanctum {
+            return Ok(false);
+        }
+
+        let token = self.mango_client.context.token_by_mint(&mint)?;
+
+        let is_lst = self.is_lst(mint);
```

Comment:
since it returns true for SOL, this isn't about lst-ness, rename to sth like `is_sanctum_supported()`